zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/CreateDocumentResult.kt

```diff
@@ -24,4 +24,24 @@ class CreateDocumentResult(
     val bestandsnaam: String,
     val bestandsomvang: Long,
     val beginRegistratie: LocalDateTime,
-)
\ No newline at end of file
+    val bestandsdelen: List<Bestandsdelen>?
```

Comment:
optioneel?

Reply:
Yes it is only returned when the inhoud parameter is kept empty in the request. 

As stated in the api description https://vng-realisatie.github.io/gemma-zaken/standaard/documenten/index 

bestandsdeel has a 0..* relation with enkelvoudiginformatieobject

Reply:
Oh I now see that it also required in the response .... lol that sounds like a mismatch, but I will make it a mandatory parameter then :)